editor/modes/visual-editor/block.js

```diff
@@ -411,7 +412,7 @@ class VisualEditorBlock extends Component {
 	}
 }
 
-export default connect(
+export default withFilters( 'VisualEditorBlock' )( connect(
```

Comment:
As we start to add multiple higher-order components, might be nice to use `flow` (`flowRight`) to compose them.

Reply:
Or expose compose as an alias of flowRight 😎

Reply:
I introduced `compose` as part of `@wordpress/element`.